src/Carbon/Translator.php

```diff
@@ -107,6 +107,16 @@ public function removeDirectory(string $directory)
         }));
     }
 
+    /**
+     * Returns the translation.
+     *
+     * @param int    $id
```

Comment:
The `$id` is casted to `string` and would nearly never be a number. So the type should just be `string`